lib/mayaUsd/ufe/UsdTransform3dFallbackMayaXformStack.cpp

```diff
@@ -143,7 +144,10 @@ void setXformOpOrder(const UsdGeomXformable& xformable)
     xformable.SetXformOpOrder(newOrder, resetsXformStack);
 }
 
-Ufe::Transform3d::Ptr createTransform3d(const Ufe::SceneItem::Ptr& item)
+Ufe::Transform3d::Ptr createEditTransform3dImp(
```

Comment:
Will add documentation.  As for input versus output, I usually stick to
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f17-for-in-out-parameters-pass-by-reference-to-non-const
so that non-const ref parameters will be modified inside the function, but I will add that information in the documentation.

Reply:
I would prefer to see more usage of Doxygen formatted documentation with proper usage of direction `[in]` and `[out]` to make this clear - https://www.doxygen.nl/manual/commands.html#cmdparam. I won't block the review on it.